cmd/infra/aws/iam.go

```diff
@@ -245,7 +245,32 @@ const (
         "arn:*:iam::*:role/*-worker-role"
       ],
       "Effect": "Allow"
-    }
+    },
```

Comment:
can we please include this in the API docs https://github.com/openshift/hypershift/blob/main/api/v1beta1/hostedcluster_types.go#L1228